app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsFragment.kt

```diff
@@ -7,23 +7,31 @@ import android.view.View
 import android.view.ViewGroup
 import org.oppia.android.app.fragment.FragmentComponentImpl
 import org.oppia.android.app.fragment.InjectableFragment
+import org.oppia.android.app.model.AdministratorControlsFragmentArguments
+import org.oppia.android.util.extensions.getProto
+import org.oppia.android.util.extensions.putProto
 import javax.inject.Inject
 
+/** Argument key  for  Administrator Controls Fragment. */
+const val ADMINISTRATOR_CONTROL_FRAGMENT_ARGUMENTS_KEY = "AdministratorControlsFragment.Arguments"
+
 /** Fragment that contains Administrator Controls of the application. */
 class AdministratorControlsFragment : InjectableFragment() {
   @Inject
   lateinit var administratorControlsFragmentPresenter: AdministratorControlsFragmentPresenter
 
   companion object {
-    private const val IS_MULTIPANE_ARGUMENT_KEY = "AdministratorControlsFragment.is_multipane"
 
     /** Creates a new instance of [AdministratorControlsFragment]. */
     fun newInstance(isMultipane: Boolean): AdministratorControlsFragment {
-      val args = Bundle()
-      args.putBoolean(IS_MULTIPANE_ARGUMENT_KEY, isMultipane)
-      val fragment = AdministratorControlsFragment()
-      fragment.arguments = args
-      return fragment
+      return AdministratorControlsFragment().apply {
+        arguments = Bundle().apply {
+          val args = AdministratorControlsFragmentArguments.newBuilder().apply {
+            this.isMultipane = isMultipane
+          }.build()
+          putProto(ADMINISTRATOR_CONTROL_FRAGMENT_ARGUMENTS_KEY, args)
+        }
+      }
```

Comment:
This can be simpliefied to:
```suggestion
     val args = AdministratorControlsFragmentArguments.newBuilder()
      .setIsMultipane(isMultipane)
      .build()
    return AdministratorControlsFragment().apply {
      arguments?.putProto(ADMINISTRATOR_CONTROL_FRAGMENT_ARGUMENTS_KEY,  args)
    }
```
Ditto everywhere else.